xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -453,6 +457,8 @@ struct RecoveryModeCursor: CursorProtocol {
     
     static func tag(_ action: ViewModel.Action) -> Model.Action {
         switch action {
+        case let .requestPresent(isPresented):
+            return .presentRecoveryMode(isPresented)
```

Comment:
Intercept `RecoveryModeAction.requestPresent` and replace it with `AppAction.presentRecoveryMode`.

RecoveryMode can fire off requests for presentation, but app owns the presentation state, since whether the recovery mode is presented is an app-level concern.